pkg/storaged/vgroup-details.jsx

```diff
@@ -94,7 +95,9 @@ class VGroupSidebar extends React.Component {
                         }));
             }
 
-            if (pvols.length === 1) {
+            if (vgroup.MissingPhysicalVolumes && vgroup.MissingPhysicalVolumes.length > 0) {
+                remove_excuse = _("Physical volumes can not be removed while a volume group is missing physical volumes.");
+            } else if (pvols.length === 1) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230531-081909-df2e1047-fedora-testing-daily//Coverage/lcov/github-pr.diff.gcov.html)